app/models/event.rb

```diff
@@ -10,5 +10,5 @@ class Event < ActiveRecord::Base
   validates :local,       presence: true
   validates :address,     presence: true
 
-  scope :not_happened, -> { where('occurred_at > ?', Time.zone.now).order(occurred_at: :asc) }
+  scope :not_happened, -> { where('occurred_at > ?', Time.current.yesterday).order(occurred_at: :asc) }
```

Comment:
Line is too long. [103/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.